docs/content/docs/connectors/datastream/kinesis.md

```diff
@@ -112,6 +112,23 @@ val kinesis = env.addSource(new FlinkKinesisConsumer[String](
     "kinesis_stream_name", new SimpleStringSchema, consumerConfig))
 ```
 {{< /tab >}}
+{{< tab "python" >}}
+```python
+consumer_config= dict() 
+consumer_config['aws.region'] = 'us-east-1'
+consumer_config['aws.credentials.provider.basic.accesskeyid'] = 'aws_access_key_id'
+consumer_config['aws.credentials.provider.basic.secretkey'] = 'aws_secret_access_key'
+consumer_config['flink.stream.initpos'] = 'LATEST'
+
+env = StreamExecutionEnvironment.get_execution_environment()
+
+JKinesisDeserializationSchema = get_gateway().jvm.org.apache.flink.streaming.connectors.
+    kinesis.serialization.KinesisDeserializationSchemaWrapper
+schema = JKinesisDeserializationSchema(SimpleStringSchema()._j_deserialization_schema)
+
+kinesis = KinesisConsumer("stream-1", KinesisDeserializationSchema(schema), consumer_config)
+```
+{{< /tab >}}
```

Comment:
Many examples in this file only have Java & Scala examples, why not update them?